app/assets/config/locale/en-US.json

```diff
@@ -30,7 +30,7 @@
     "feedback": "Feedback",
     "forum": "Help Forum",
     "forumLink": "https://discuss.nebula-graph.io/",
-    "ask": "Sure to do this?",
+    "ask": "Are you sure to proceed?",
```

Comment:
---> Are you sure you want to proceed?
Are you sure to do means do you ensure the action(to do) instead of what you intend.

Reply:
Thank you @abby-cyber, I think need your expertise to help review/revisit all i18n fields of the studio, I was just trying to do some of them(in the wrong way as you help pointed here ;-) ), would you mind helping when you have the bandwidth to do so?

Reply:
yeah, sure